Makefile

```diff
@@ -10,34 +12,41 @@ dep = $(obj:.o=.d)  # one dependency file for each source
 #macos = y
 #debug = y
 #seq = y
-
-CFLAGS = -DMKL_ILP64 -m64 -I${MKLROOT}/include -I${NUMAROOT}/include -I${TBBROOT}/include -mavx2 -mfma -mf16c -fopenmp -mavx512f -Wall #-march=skylake
+SP ?=1
+UBN ?=0
```

Comment:
So USE_BROADCAST_NODE doesn't give speedup right?

Reply:
If there was an improvement, it was very small. It basically removes the overhead of spawning a task for the very first node only.